TTHAnalysis/cfg/run_susyDeDx_cfg.py

```diff
@@ -9,59 +9,42 @@
 from CMGTools.RootTools.samples.configTools import *
 from CMGTools.RootTools.samples.autoAAAconfig import *
 
-#-------- Standard susy setup (useful to get core modules)
-from CMGTools.TTHAnalysis.analyzers.susyCore_modules_cff import *
+from CMGTools.TTHAnalysis.analyzers.xtracks_modules_cff import *
 
-lepAna.doMiniIsolation = "precomputed"
-lepAna.mu_isoCorr = "deltaBeta"
-lepAna.loose_muon_id     = "POG_ID_Loose"
-lepAna.loose_electron_id = "MVA_ID_nonIso_Fall17_Loose"
-lepAna.loose_muon_isoCut     = lambda muon : muon.miniRelIso < 0.4 and muon.sip3D() < 8
-lepAna.loose_electron_isoCut = lambda elec : elec.miniRelIso < 0.4 and elec.sip3D() < 8
+lepAna.loose_muon_isoCut     = lambda muon : muon.relIso03 < 0.25
+lepAna.loose_electron_isoCut = lambda elec : elec.relIso03 < 0.25
 
 tauAna.loose_ptMin = 20
 tauAna.loose_etaMax = 2.3
 tauAna.loose_tauID = "decayModeFindingNewDMs"
-tauAna.loose_vetoLeptons = False # no cleaning with leptons in production
-
-photonAna.do_mc_match = False
+tauAna.loose_vetoLeptons = False 
 
 jetAna.addJECShifts = True
 jetAna.jetPtOrUpOrDnSelection = True
 
-jetAna.copyJetsByValue = True # do not remove this
-metAna.copyMETsByValue = True # do not remove this
+jetAna.copyJetsByValue = True 
 jetAna.calculateType1METCorrection = True
-metAna.recalibrate = "type1"
 jetAnaScaleUp.calculateType1METCorrection = True
-metAnaScaleUp.recalibrate = "type1"
 jetAnaScaleDown.calculateType1METCorrection = True
-metAnaScaleDown.recalibrate = "type1"
 
-## early skimming with loose cuts (before jet-lepton cleaning, re-application of JECs, ...)
-from CMGTools.TTHAnalysis.analyzers.ttHFastMETSkimmer import ttHFastMETSkimmer
-fastMETSkim = cfg.Analyzer( ttHFastMETSkimmer, name='fastMETSkimmer',
-    met      = "slimmedMETs", # met collection to use
-    metCut    =  50,  # MET cut, looser because of non-final JECs
-    )
-from CMGTools.TTHAnalysis.analyzers.ttHFastJetSkimmer import ttHFastJetSkimmer
-fastJetSkim = cfg.Analyzer(ttHFastJetSkimmer, name="fastJetSkimmer",
-    jets = 'slimmedJets',
-    jetCut = lambda j : j.pt() > 80 and abs(j.eta()) < 2.4, # looser pt because of non-final JECs
-    minJets = 1,
-    )
```

Comment:
these were there for cpu efficiency reasons (it's faster to reject events earlier in the processing)
did you check what is the impact of removing them?

Reply:
No, I did not check ... the jet skim we can keep, but the MET one we should get rid of. Cutting on MET could bias METNoMu which is what we would like to cut on. I don't know if there is an easy way to do a pre-filtering on METNoMu (since it is produced later on)